### ISSUE TITLE
[Bug Report][3.6.14] v-tree-view warning: Failed to resolve directive: ripple

### ISSUE THREAD

**Issue by tzfun** [vuetifyjs/vuetify#20299]
### Environment
**Vuetify Version:** 3.6.14
**Vue Version:** 3.3.4
**Browsers:** Chrome 127.0.0.0
**OS:** Windows 10

### Steps to reproduce
Just use v-tree-view
```vue
<v-treeview
    v-model:selected="treeValue"
    :items="treeData"
    :opened="initiallyOpen"
    item-key="name"
    activatable
    selectable
    open-strategy="multiple"
    return-object
    select-strategy="independent"
    open-on-click
    density="compact"
>
  <template v-slot:prepend="{ item, open }">
    <v-icon v-if="!item.file">
      {{ open ? 'mdi-folder-open' : 'mdi-folder' }}
    </v-icon>
    <v-icon v-else>
      {{ fileIcon[item.file] }}
    </v-icon>
  </template>
</v-treeview>
```

```
[Vue warn]: Failed to resolve directive: ripple 
  at <VTreeviewItem title="static" value=undefined onClick= 
```

### Expected Behavior
No warning

### Actual Behavior
Whenever the item is rendered or the mouse is hovered over, warnings are printed crazily. There are so many warnings that the console crashes.

### Reproduction Link
[https://github.com/tzfun/etcd-workbench](https://github.com/tzfun/etcd-workbench)

**Comment by tzfun** [comment]
This is my warning stack, hope it helps

![image](https://github.com/user-attachments/assets/8093e044-a17c-4f79-a400-4ce2b2b55d8b)

treeData:
```ts
const treeData = ref([
  {
    id: 2,
    title: 'node_modules',
  },
  {
    id: 3,
    title: 'public',
    children: [
      {
        id: 4,
        title: 'static',
        children: [{
          id: 5,
          title: 'logo.png',
          file: 'png',
        }],
      },
      {
        id: 6,
        title: 'favicon.ico',
        file: 'png',
      },
      {
        id: 7,
        title: 'index.html',
        file: 'html',
      },
    ],
  },
  {
    id: 12,
    title: 'vue.config.js',
    file: 'js',
  },
  {
    id: 13,
    title: 'yarn.lock',
    file: 'txt',
  },
])
```

**Comment by yuwu9145** [comment]
It happens when `activatable` is used.

Tested it has been resolved in `3.7.0-beta.1`
